cnxarchive/sql/migrations/20160622095217_add_modules_stateid_default.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+
+def up(cursor):
+    cursor.execute("""\
+ALTER TABLE modules
+  ALTER COLUMN stateid SET DEFAULT 1""")
+
```

Comment:
Hmm, this might take an exclusive lock on the table to set the default. I'll have to check.